pygmt/base_plotting.py

```diff
@@ -919,15 +920,20 @@ def logo(self, **kwargs):
         box : bool or str
             Without further options, draws a rectangular border around the
             GMT logo.
+        style : str
+            ``l|n|u``.
+            Control what is written beneath the map portion of the logo.
+
+            - **l** to plot the text label “The Generic Mapping Tools”
```

Comment:
```suggestion
            - **l** to plot the text label "The Generic Mapping Tools"
              [Default]
```
Just one small suggestion.